smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HTTPBindingProtocolGenerator.kt

```diff
@@ -390,6 +390,10 @@ abstract class HTTPBindingProtocolGenerator(
         }
     }
 
+    override fun generateSmokeTests(ctx: ProtocolGenerator.GenerationContext) {
+        return SmokeTestGenerator(ctx).generateSmokeTests()
+    }
```

Comment:
This concrete implementation generates smoke tests according to generic Smithy spec.